docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -302,7 +304,7 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | input.application.excludeNamespaces | []string | List of Namespaces to exclude during log collection from all Namespaces (mutually exclusive with `namespaces`) |
 | input.application.containers | []string | List of containers to collect from (mutually exclusive with `excludeContainers`) |
 | input.application.excludeContainers | []string | List of containers to exclude (mutually exclusive with `containers`) |
-| input.application.includeSystemNamespaces | boolean | If you specify neither `namespaces` nor `excludeNamespaces`, you can set `includeSystemNamespaces` to include all system Namespaces without listing them manually. Defaults to `false`. |
+| input.application.includeSystemNamespaces | boolean | If you specify neither `namespaces` nor `excludeNamespaces`, you can set `includeSystemNamespaces` to include all system Namespaces without listing them manually. Defaults to `false`. If enabled, it is recommended to exclude the logs from the Fluent Bit container to avoid problems with recursive logs, see also [Create a LogPipeline Input](#step-2-create-an-input) |
```

Comment:
```suggestion
| input.application.includeSystemNamespaces | boolean | If you specify neither `namespaces` nor `excludeNamespaces`, you can set `includeSystemNamespaces` to include all system Namespaces without listing them manually. Defaults to `false`. If enabled, it is recommended to exclude the logs from the Fluent Bit container to avoid problems with recursive logs. See also [Create a LogPipeline Input](#step-2-create-an-input). |
```